doc/en/developer/source/qa-guide/index.rst

```diff
@@ -140,11 +158,29 @@ Spotbugs
 
 The `Spotbugs <https://spotbugs.github.io/>`_ checker runs as a post-compile bytecode analyzer.
 
+The plugin version is managed in the pluginManagement section:
+
+.. code-block:: xml
+
+   <pluginManagement>
+     <plugins>
+       ...
+       <plugin>
+         <groupId>com.github.spotbugs</groupId>
+         <artifactId>spotbugs-maven-plugin</artifactId>
+         <version>${spotbugs-maven-plugin.version}</version>
+       </plugin>
+       ...
+     </plugins>
+   </pluginManagement>
```

Comment:
Q: Does adding this by hand defeats the purpose of literal include below (ie having the docs being live from the pom). Perhaps it is not so useful to have?

The raw configuration I guess is the useful part, just the sentence "The plugin version is managed in pluginMangaement section". I sufficient.